performance/collector/es.py

```diff
@@ -0,0 +1,89 @@
+
```

Comment:
Do not forget to include Apache 2 license header: 
https://github.com/seecloud/performance/blob/master/performance/config.py#L1-L14